### PR TITLE
[DOC] Update documents

### PR DESCRIPTION
**Pull request by PHILO-HE**
Update build option, function support states, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,16 +156,15 @@ Gluten is licensed under [Apache 2.0 license](https://www.apache.org/licenses/LI
 
 # 7 Contact
 
-Gluten was initiated by Intel and Kyligence in 2022. Several companies such as Intel, Kyligence, BIGO, Meituan, Alibaba Cloud, NetEase, Baidu, Microsoft and others, are actively participating in the development of Gluten. If you are interested in Gluten project, please contact below email address for further discussion.
+Gluten was initiated by Intel and Kyligence in 2022. Several other companies are also actively participating in the development, such as BIGO, Meituan, Alibaba Cloud, NetEase, Baidu, Microsoft, etc. If you are interested in Gluten project, please contact and subscribe below email address for further discussion.
 
-binwei.yang@intel.com; weiting.chen@intel.com;
-chang.chen@kyligence.io; zhichao.zhang@kyligence.io; neng.liu@kyligence.io;
-zuochunwei@meituan.com;yangchuan.zy@alibaba-inc.com;xiyu.zk@alibaba-inc.com;joey.ljy@alibaba-inc.com
+dev@gluten.apache.org for community activity
```
Comment (jackylee-ch) [inline README.md:161]: Since we have the concat email here, do we sill need the `concat-us.md`?

Comment (PHILO-HE) [reply README.md:161]: I think we can remove that doc. Will remove that.

**Comment by PHILO-HE** [comment]
@weiting-chen, could you take a look? Thanks!
Also cc @jackylee-ch if you have other comment.

**Review by rui-mo — APPROVED** [review]
Thanks!

**Review by jackylee-ch — APPROVED** [review]
LGTM

**Pull request merged**